airflow/api_connexion/endpoints/forward_to_fab_endpoint.py

```diff
@@ -79,12 +79,14 @@ def delete_role(**kwargs) -> APIResponse:
 @_require_fab
 def patch_role(**kwargs) -> APIResponse:
     """Update a role."""
+    kwargs.pop("body", None)
```

Comment:
Where this `body` come from? Why dont we have the issue when we test `role_and_permission_endpoint.patch_role(**kwargs)` directly?

Reply:
I am saying this because here this function only forwards args to `role_and_permission_endpoint.patch_role`, so whatever issue we have here, we should then have the same one at the destination

Reply:
If you test the endpoints: `api/v1/roles` and `api/v1/users` with post/patch calls they will fail with the above errors.

Kwargs contains all your request args which are now forwarded to the `role_and_permission_endpoint.patch_role`. That's where the body is coming from. it's the request.json

You can test `api/v1/roles` and `api/v1/users` directly in the current main to see the error.

Reply:
I see. LGTM